kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -1,12 +1,16 @@
 <template>
 
   <div class="dropdown">
-    <button
+    <icon-button
```

Comment:
We can leave this, but I'm not totally convinced that using `icon-button` here makes sense. A quick brain dump follows:

It's basically a trade-off between keeping styling DRY in general, and keeping our global components as simple and easy-to-maintain as possible. The more configurable our global components are, the higher a chance that we'll make changes which introduce regressions because it's hard to see and rationalize every possible state they can take - plus all options need to be supported forever. Adding config options also weakens a component's power as a 'style guide enforcer'.

DRY is generally a good thing, but comes with a complexity cost. "How hard will it be to maintain these things separately?" vs "What risks of developer comprehension and API complexity are introduced?" 

Here's well-written blog post on the topic that I think is worth a read: http://thereignn.ghost.io/on-dry-and-the-cost-of-wrongful-abstractions/